package.yaml

```
@@ -1,5 +1,5 @@
 name: stack-lint-extra-deps
-version: 1.2.3.1
+version: 1.2.4.1
```

Your changelog looks like this was meant to be

```suggestion
version: 1.2.4.0
```

---

Good catch.